src/Halcyon/Datasource/DbDatasource.php

```diff
@@ -90,27 +86,22 @@ public function getQuery($ignoreDeleted = true)
     }
 
     /**
-     * Helper to make file path.
+     * Helper method to make the full file path to the model.
```

Comment:
"the full" implies the absolute path to the file on the filesystem, which is incorrect for databases and may not be correct for the filesystem source either (haven't looked). Thoughts?

Reply:
Perhaps "Helper method to make the full relative file path to the model"? I just thought the original comment was not descriptive enough.